{"uuid": "20341268-d840-44ef-9cbc-d7f55b952432", "children": ["37501dd6-da0d-48ea-bac1-3661cb6b0658", "64f043aa-d447-4a14-87fb-7ce5ca8f6c78", "88fa6f5d-891a-436c-b65a-dbd96b839ca1", "e2afb354-2dd9-46a9-b5d7-abf0aaeea454", "33c8fd42-dfab-4ed2-af02-9b92b1a2d4cc"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/test_api.py\", line 30, in setup\n    peviitor_jobs_data = TestScrapers.setup_tests.get_jobs_careers(scraper_class)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/test_api.py\", line 17, in get_jobs_careers\n    self.scraper_data = self.jobdetails.scrape_peviitor(scraper_class)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/job_details_utils.py\", line 91, in scrape_peviitor\n    response_data = JobDetails._get_request(company_name)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/job_details_utils.py\", line 47, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/peviitor_update.py\", line 38, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.laurentiumarian.ro/get_token\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1734153684448, "stop": 1734153684653}], "start": 1734153684448, "stop": 1734153684817}